### ISSUE TITLE
bucardo is failing to sync tables with long identifiers

### ISSUE THREAD

**Issue by josh-janrain** [bucardo/bucardo#125]
```
10:09 < jwillhite> reproduced the error: Failed : DBD::Pg::st execute failed: ERROR:  relation "bucardo.delta_484en3x9dn9ykkn3dcrvnn2guz_cocacola_user_164!75e475389f" does not exist LINE 3:
10:10 < jwillhite> looking in source bucardo schema this is what I see: 
10:10 < jwillhite> capture-dev=> \d bucardo."delta_484en3x9dn9ykkn3dcrvnn2guz_co
10:10 < jwillhite> bucardo."delta_484en3x9dn9ykkn3dcrvnn2guz_cocacola_user_1647_!cfae0508ba"  bucardo."delta_484en3x9dn9ykkn3dcrvnn2guz_coke_policy_checker!1b876d0f47"  
10:10 < jwillhite> bucardo."delta_484en3x9dn9ykkn3dcrvnn2guz_coke_policy_checker!0c79232f0e"  bucardo."delta_484en3x9dn9ykkn3dcrvnn2guz_coke_policy_checker!580d40d902"  
10:11 < jwillhite> something weird going on with how the tables are getting named,  you can see the table that bucardo is complaining about exists but has a longer name in the source schema
10:28 < endpoint_david> yeah, interesting, that's a good clue
10:32 < endpoint_david> jwillhite: what are the full names of the tables?
10:32 < endpoint_david> the non-delta versions
10:32 -!- Irssi: Pasting 7 lines to #bucardo. Press Ctrl-K if you wish to do this or Ctrl-C to cancel.
10:32 < jwillhite> capture-dev=> \d "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647
10:32 < jwillhite> "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647                              "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history                      "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history_pkey
10:32 < jwillhite> "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_count                        "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history_entity_uuid_idx      
                   "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history_timestamp_idx
10:32 < jwillhite> "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_count_id_seq                 "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history_list                 "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_pkey
10:32 < jwillhite> "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_count_pkey                   "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history_list_id_seq          "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_uuid_key
10:32 < jwillhite> "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_created_idx                  "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history_list_pkey            
10:32 < jwillhite> "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_email_1655_check_unique      "484en3x9dn9ykkn3dcrvnn2guz".cocacola_user_1647_history_list_table_name_key 
```

**Comment by machack666** [comment]
IRC discussion indicates this issue is related to a leading int in the schema name.

**Comment by machack666** [comment]
fixed in 935caaaa6a015bd742d8d03e5c5ea1283db5dd2b.
